docs/index.md

```diff
@@ -71,3 +71,6 @@ Agendas and post-meeting notes.
 ## Cylc 8 Guides
 Guides for developers, and therefore slightly separate to main documentation
 - [Getting Started](howto/environment.md)
+
+## Misc
+- [thread modeling outcomes](threats.md)
```

Comment:
s/thread/threat

Reply:
that was quick :grin: thanks will fix...